ngraph_creator/operations/src/ResizeBilinear.cpp

```diff
@@ -0,0 +1,199 @@
+        ALOGD("%s Forced NCHW conversion at operationIndex %d", __func__, mNnapiOperationIndex);
+    }
+
+    std::vector<float> scaleFactors = {1.0f, 1.0f, width_scale, height_scale};
+    std::shared_ptr<ngraph::Node> scaleConstNode, shape;
+    // TODO: VPU Plugin may not support all type of conversions. Update this
+    // section to support VPUX Plugin
+    if (checkInputOperandType(1, (int32_t)OperandType::FLOAT16)) {
+        shape = std::make_shared<ngraph::opset3::Convert>(
+            std::make_shared<ngraph::opset3::ShapeOf>(inputNode), ngraph::element::f16);
+        scaleConstNode = createConstNode(ngraph::element::f16, {4}, scaleFactors);
+    } else if (checkInputOperandType(1, (int32_t)OperandType::FLOAT32) ||
+               checkInputOperandType(1, (int32_t)OperandType::INT32)) {
+        shape = std::make_shared<ngraph::opset3::Convert>(
+            std::make_shared<ngraph::opset3::ShapeOf>(inputNode), ngraph::element::f32);
+        scaleConstNode = createConstNode(ngraph::element::f32, {4}, scaleFactors);
+    }
+    // Scale input shape to calculate final output shape
+    auto outputShape = std::make_shared<ngraph::op::v1::Multiply>(scaleConstNode, shape);
+
+    // TODO: VPU Plugin may not support all type of conversions. Update this
+    // section to support VPUX Plugin
+    auto outputShape_int32 =
+        std::make_shared<ngraph::opset3::Convert>(outputShape, ngraph::element::i32);
+
+    auto begin_index = createConstNode(ngraph::element::i32, {1}, convertToVector(1));
+    auto end_index = createConstNode(ngraph::element::i32, {1}, convertToVector(4));
+    auto strides = createConstNode(ngraph::element::i32, {1}, convertToVector(2));
+
+    int64_t begin_mask = 0;
+    int64_t end_mask = 0;
+    // Get output shape(width x height) from scaled Shape vector
+    auto outputShapeNode = std::make_shared<ngraph::op::v1::StridedSlice>(
+        outputShape_int32, begin_index, end_index, strides, convertToVector(begin_mask),
+        convertToVector(end_mask));
+
+    std::vector<float> scale_vec = {width_scale, height_scale};
+    std::shared_ptr<ngraph::Node> scaleNode;
+    if (checkInputOperandType(1, (int32_t)OperandType::FLOAT16))
+        scaleNode = createConstNode(ngraph::element::f16, {2}, scale_vec);
+    else if (checkInputOperandType(1, (int32_t)OperandType::FLOAT32) ||
+             checkInputOperandType(1, (int32_t)OperandType::INT32)) {
+        scaleNode = createConstNode(ngraph::element::f32, {2}, scale_vec);
+    }
+
+    std::vector<int32_t> axes_vec = {2, 3};
+    auto axesNode = createConstNode(ngraph::element::i32, {2}, axes_vec);
+
+    outputNode = std::make_shared<ngraph::op::v4::Interpolate>(inputNode, outputShapeNode,
+                                                               scaleNode, axesNode, attrs);
+    if (!useNchw) {
+        outputNode = transpose(NCHW_NHWC, outputNode);
+        mNgraphNodes->setForcedNchw(mDefaultOutputIndex, false);
```

Comment:
You may have to remove this too, the forcedNCHW like the inputnode.

Reply:
i removed it in the second commit

Reply:
Will update this. Thanks